kornia/morphology/basic_operators.py

```diff
@@ -0,0 +1,127 @@
+# Libraries
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+
+# se_to_mask
+def se_to_mask(se: torch.Tensor) -> torch.Tensor:
+    se_h, se_w = se.size()
+    se_flat = se.view(-1)
+    num_feats = se_h * se_w
+    out = torch.zeros(num_feats, 1, se_h, se_w, dtype=se.dtype, device=se.device)
+    for i in range(num_feats):
+        y = i % se_h
+        x = i // se_h
+        out[i, 0, x, y] = (se_flat[i] >= 0).float()
+    return out
+
+
+# dilation
+class Dilate(nn.Module):
+
+    def __init__(self, se: torch.Tensor) -> None:
+        super().__init__()
+        self.se = se - 1
+        self.se_h, self.se_w = se.shape
+        self.pad = (self.se_h // 2, self.se_w // 2)
+        self.kernel = se_to_mask(self.se)
+
+    def forward(self, input: torch.Tensor) -> torch.Tensor:
+        output = input.view(input.shape[0] * input.shape[1], 1, input.shape[2], input.shape[3])
+        output = (F.conv2d(output, self.kernel, padding=self.pad) + self.se.view(1, -1, 1, 1)).max(dim=1)[0]
+
+        return output.view(*input.shape)
+
+
+def dilation(tensor: torch.Tensor, kernel: torch.Tensor) -> torch.Tensor:
+
+    r"""
+    Returns the dilated image applying the same kernel in each channel.
```

Comment:
add an end line between the main functon description and details. Same for other functions